mne/io/edf/edf.py

```diff
@@ -87,9 +87,10 @@ class RawEDF(BaseRaw):
         Names of channels or list of indices that should be designated
         MISC channels. Values should correspond to the electrodes in the
         edf file. Default is None.
-    stim_channel : False
-        If False, there will be no stim channel added from a TAL channel.
-        None is accepted as an alias for False.
+    stim_channel : 'auto' | list of str
```

Comment:
so int indexing is supported now or not?

Reply:
yes. I updated the docstring